src/pages/leeszaal/[readingRoomSlug]/[objectId]/index.tsx

```diff
@@ -26,32 +26,66 @@ import { createPageTitle } from '@shared/utils';
 import {
 	DynamicActionMenu,
 	Metadata,
-	MetadataItem,
-	// ObjectPlaceholder,
+	ObjectPlaceholder,
 	RelatedObject,
 	RelatedObjectProps,
 } from 'modules/media/components';
-import { metadataMock } from 'modules/media/components/Metadata/__mocks__/metadata';
-// import { objectPlaceholderMock } from 'modules/media/components/ObjectPlaceholder/__mocks__/object-placeholder';
+import { objectPlaceholderMock } from 'modules/media/components/ObjectPlaceholder/__mocks__/object-placeholder';
 
 const ObjectDetailPage: NextPage = () => {
 	/**
 	 * Hooks
 	 */
-	const [activeTab, setActiveTab] = useState<string | number>(ObjectDetailTabs.Media);
-	const [pauseMedia, setPauseMedia] = useState(true);
 	const { t } = useTranslation();
 	const router = useRouter();
-	const windowSize = useWindowSizeContext();
+
+	// Internal state
+	const [activeTab, setActiveTab] = useState<string | number | undefined>(undefined);
+	const [mediaType, setMediaType] = useState<MediaTypes>(null);
+	const [pauseMedia, setPauseMedia] = useState(true);
+
+	// Layout
 	useStickyLayout();
 	useNavigationBorder();
 	useHideFooter();
 
+	// Sizes
+	const windowSize = useWindowSizeContext();
 	const showNavigationBorder = useSelector(selectShowNavigationBorder);
 
 	const metadataRef = useRef<HTMLDivElement>(null);
 	const metadataSize = useElementSize(metadataRef);
 
+	// Fetch data
+	const { data: mediaInfo, isLoading: isLoadingMediaInfo } = useGetMediaInfo(
+		router.query.objectId as string
+	);
+
+	// Set default view
+	useEffect(() => {
+		setMediaType(mediaInfo?.dctermsFormat as MediaTypes);
+
+		if (windowSize.width && windowSize.width < 768) {
+			// Default to metadata tab on mobile
+			setActiveTab(ObjectDetailTabs.Metadata);
+		} else {
+			// Check media content for default tab on desktop
+			setActiveTab(
+				mediaInfo?.dctermsFormat ? ObjectDetailTabs.Media : ObjectDetailTabs.Metadata
+			);
+		}
+		// eslint-disable-next-line react-hooks/exhaustive-deps
+	}, [mediaInfo]);
+
+	useEffect(() => {
+		if (mediaType === null) {
+			if (windowSize.width && windowSize.width >= 768) {
+				// Always expand metadata on desktop if media is not defined
+				setActiveTab(ObjectDetailTabs.Metadata);
+			}
+		}
+	}, [windowSize, mediaType]);
```

Comment:
Je kan dit vermijden door de "expanded" styling in een media query te steken, open of dicht op desktop maakt dan geen verschil. (Zie [SidebarLayout](https://github.com/viaacode/hetarchief-client/pull/178/files#diff-aa95d9b95c27d0a1c5de0a3611b9525d070fb0869013f3150fc15ea98b834b7bR68-R69))